Model/Method/Afterpay.php

```diff
@@ -593,8 +593,8 @@ public function updateShippingAddressByDpdParcel($quote, &$requestData)
         $matches = false;
         if ($fullStreet && preg_match('/(.*)\s([0-9]+)([^\w]*)([\w]*)/', $fullStreet, $matches)) {
             $street = $matches[1];
-            $streetHouseNumber = $matches[2];
-            $streetHouseNumberSuffix = @$matches[4];
+            $streetHouseNumber = $matches[2];            
+            $streetHouseNumberSuffix = isset($matches[4]) ? $matches[4] : '';
```

Comment:
what about usage of null coalescing operator (??) to be more compact?

Reply:
@vladislav-padalka-hys sure, good idea.